### PR TITLE
Add installation guide to molecular nodes

### PR DESCRIPTION
**Pull request by yuxuanzhuang**
TODO
- add youtube link after someone has uploaded it to MDA channel.

### CODE REVIEW

**Comment by IAlibay** [comment]
@yuxuanzhuang could you explain what you mean by the YouTube link? I think I've missed part of the conversation, is there a video somewhere that we should be uploading?

**Comment by yuxuanzhuang** [comment]
@orbeckst has offered to upload this short tutorial video about how to get started to the MDA channel https://drive.google.com/file/d/1LHrQ-wFJ_Xzf-0vi2B4CfxYKg9TbZycf/view

**Comment by IAlibay** [comment]
> @orbeckst has offered to upload this short tutorial video about how to get started to the MDA channel https://drive.google.com/file/d/1LHrQ-wFJ_Xzf-0vi2B4CfxYKg9TbZycf/view

Thanks, I'll check up on the status and upload it if necessary.

**Comment by IAlibay** [comment]
@yuxuanzhuang is there a video title and description you would like?

**Comment by yuxuanzhuang** [comment]
Video Title:

How to Set Up MolecularNodes and Blender Jupyter Kernel for UGM 2023

Description:

Learn the installation process by following the instructions at https://github.com/MDAnalysis/UGM2023/blob/main/hackathon/blender/install.md

**Comment by IAlibay** [comment]
@yuxuanzhuang here is the video link: https://youtu.be/Lmm7n2ie4rM

File: hackathon/blender/README.md [hunk 1]
```diff
@@ -12,34 +12,34 @@ of MD trajectories. Under the hood, *Molecular Nodes* uses MDAnalysis for trajec
...
 molecular dynamics trajectories, `MDAnalysis` has been used as the interface enabling import
-in to Blender.
+into Blender.
 
-The current implementations reads all of the requested frames of the trajectory in to
-memory, and discards the MDAnalysis session after import is complete.
+The current implementations read all of the requested frames of the trajectory into
+memory, and discard the MDAnalysis session after importing is complete.
 
 ### A live MDAnalysis session
```
Comment (IAlibay) [inline hackathon/blender/README.md:30]: Did you want to put a section about interacting over the hackathon discord somewhere?

**Comment by yuxuanzhuang** [comment]
> @yuxuanzhuang here is the video link: https://youtu.be/Lmm7n2ie4rM

Thanks!

**Review by IAlibay — APPROVED** [review]
lgtm, thanks

**Pull request merged**